packages/material-ui/src/Stepper/Stepper.test.tsx

```diff
@@ -239,17 +240,23 @@ describe('<Stepper />', () => {
   it('should be able to force a state', () => {
     const { container } = render(
       <Stepper>
-        <Step />
-        <Step active />
-        <Step />
+        <Step>
+          <StepLabel>one</StepLabel>
+        </Step>
+        <Step active>
+          <StepLabel>two</StepLabel>
+        </Step>
+        <Step>
+          <StepLabel>tree</StepLabel>
+        </Step>
       </Stepper>,
     );
 
-    const steps = container.querySelectorAll(`.${stepClasses.root}`);
+    const steps = container.querySelectorAll(`.${stepIconClasses.root}`);
 
-    expect(steps[0]).not.to.have.class(stepClasses.active);
-    expect(steps[1]).not.to.have.class(stepClasses.active);
-    expect(steps[2]).not.to.have.class(stepClasses.active);
+    expect(steps[0]).to.have.class(stepIconClasses.active);
+    expect(steps[1]).to.have.class(stepIconClasses.active);
+    expect(steps[2]).not.to.have.class(stepIconClasses.active);
```

Comment:
`stepClasses.active === undefined`

Reply:
Why would two icons have the active class when only one step is active?

Reply:
The test case renders as:

<img width="580" alt="Capture d’écran 2021-06-22 à 18 06 00" src="https://user-images.githubusercontent.com/3165635/122961264-8676ab80-d384-11eb-8d16-37499226a2c8.png">

The first icon is active because activeStep default to 0, and the second one is active because of the extra prop.

Reply:
We could remove the test too, not real preferences. My intent was to have the test file migrated to TS, and then, noticed this useless test. This is an attempt to make it meaningful.

Reply:
but only one step can be active. What you're showing is "completed".

Reply:
OK, I think that we are going too off topic to the problem the PR is trying to solve. I will remove this dead test case, without adding a new one.

Reply:
For context the initial test we added is identical to the new one I'm proposing https://github.com/mui-org/material-ui/pull/13188.

I would also ask to reconsider your perspective on the problem based on the reactions that the developers had on your past answers.

I personally do think that multiple active steps make sense. When the steps are small enough, the information can be displayed all at once.

Reply:
> I would also ask to reconsider your perspective on the problem based on the reactions that the developers had on your past answers on this topic

Why would I do that?

> I personally do think that multiple active steps make sense. When the steps are small enough, the information can be displayed all at once.

Then don't use a stepper. These have a very specific use case bases on user research. You're defeating the purpose of a stepper by displaying multiple steps.

Reply:
Are you OK with removing this test? https://github.com/mui-org/material-ui/pull/26874#discussion_r657297334